src/common/feedGenerator.ts

```diff
@@ -162,23 +163,28 @@ export const feedToFilters = async (
     { includeTags: [], blockedTags: [] },
   );
 
-  const { excludeTypes, blockedContentCuration } = settings.reduce<{
-    excludeTypes: string[];
-    blockedContentCuration: string[];
-  }>(
-    (acc, curr) => {
-      if (curr.options?.type) {
-        if (curr.group === 'content_types') {
-          acc.excludeTypes.push(curr.options.type);
-        }
-        if (curr.group === 'content_curation') {
-          acc.blockedContentCuration.push(curr.options.type);
+  const { excludeTypes, blockedContentCuration, excludeSourceTypes } =
+    settings.reduce<{
+      excludeTypes: string[];
+      blockedContentCuration: string[];
+      excludeSourceTypes: string[];
```

Comment:
We need to add this to `FiltersInput` gql type?

Reply:
They actually never really come from frontend though, but think we might use it via internal graph resolver on API